packages/components/src/components/modal/modal-styles.ts

```diff
@@ -2,23 +2,27 @@ import type { JssStyle } from 'jss';
 import type { GetJssStyleFunction } from '../../utils';
 import { BREAKPOINTS, buildResponsiveStyles, buildSlottedStyles, getCss, mergeDeep, parseJSON } from '../../utils';
 import type { BreakpointCustomizable, BreakpointKey } from '../../types';
-import { gridSafeZone, headingMediumStyle, getMediaQueryMin } from '@porsche-design-system/utilities-v2';
+import {
+  gridSafeZone,
+  getMediaQueryMin,
+  borderRadiusMedium,
+  frostedGlassStyle,
+  borderWidthBase,
+  headingLargeStyle,
+} from '@porsche-design-system/utilities-v2';
 import {
   addImportantToEachRule,
   getBaseSlottedStyles,
-  getFocusJssStyle,
   getInsetJssStyle,
   getThemedColors,
   pxToRemWithUnit,
 } from '../../styles';
 import { MODAL_Z_INDEX } from '../../constants';
-import { getFocusVisibleFallback } from '../../styles/focus-visible-fallback';
+import { themeDarkBackgroundShading } from '@porsche-design-system/utilities-v2/src/js/theme/themeDarkBackgroundShading';
```

Comment:
Shouldn't be imported from source but from built package instead.
Perhaps linting rules need to be adjusted?

Reply:
`themeDarkBackgroundShading` is not included into the build atm, because it's for internal usage only.

Reply:
`themeDarkBackgroundShading` is currently not included into the build because it is for internal usage only (I guess).

Reply:
I've added it to `utilities` package and changed the import.